src/main/java/net/smartcosmos/dto/metadata/MetadataCreate.java

```diff
@@ -0,0 +1,37 @@
+package net.smartcosmos.dto.metadata;
+
```

Comment:
@wfschneider-rfid I'm not that sure anymore: Isn't it possible that the response models appear in the generated documentation if they aren't converted again in the REST layer?

For this case, any potential response model probably might get the Swagger annotations.